### ISSUE TITLE
Add support for expand

### ISSUE THREAD

**Issue by baelen-git** [cgascoig/isctl#38]
Please see 

https://intersight.com/apidocs/introduction/query/#expand-query-option-returning-related-resources

Something similar as the --filter option, but than for --expand

**Comment by cgascoig** [comment]
Good suggestion. This might make sense along with or instead of #7.

**Comment by cgascoig** [comment]
Added in v0.1.9

```
❯ isctl get kubernetes version --name 1.21.10-iks.0 --expand 'OvaImageTemplate($select=Name)'
INFO[0001] Single result, falling back to vertical output. NOTE: this is not valid YAML; use --output yaml to get valid YAML.
BootIso: null
Catalog: MoRef[kubernetes.Catalog/60627c1f7a6f722d30785b23]
KubernetesVersion: v1.21.10
Moid: 623d0cf97a6f722d30c29fcd
Name: 1.21.10-iks.0
OvaImageTemplate:
  ClassId: software.SolutionDistributable
  Moid: 623d069b6567612d31935f2a
  Name: ki-20220315-0-1.21.10-ubuntu20-iks-vsphere.ova
  ObjectType: software.SolutionDistributable
Qcow2NodeTemplate: null
Qcow2VirtualMachineTemplate: MoRef[software.SolutionDistributable/623d01606567612d3192f8f0]
```

**Comment by baelen-git** [comment]
You are fantastic! Tried this out and works like a charm.
Even works with nested $expands.


```bash

❯ isctl get kubernetes clusterprofile -o yaml --select 'Name,NodeGroups' --expand 'NodeGroups($select=KubernetesVersion;$expand=KubernetesVersion($select=Version;$expand=Version($select=Name,Tags)))' | egrep -v "ObjectType|ClassId"

```

```yaml

  Moid: 61af52617a6f722d31bd3d84

  Name: baelen-azure-arc

  NodeGroups:

    KubernetesVersion:

      Moid: 61f3bc387a6f722d31a3090a

      Version:

        Moid: 61f381e17a6f722d3093c093

        Name: 1.20.14-iks.0

        Tags:

        - Key: intersight.kubernetes.SupportStatus

          Value: UpgradeRequired

    Moid: 61af55bb7a6f722d31bdada4

    KubernetesVersion:

      Moid: 61f3bc387a6f722d31a3090a

      Version:

        Moid: 61f381e17a6f722d3093c093

        Name: 1.20.14-iks.0

        Tags:

        - Key: intersight.kubernetes.SupportStatus

          Value: UpgradeRequired

    Moid: 61af55cc7a6f722d31bdaf58

  ```



From: Chris Gascoigne ***@***.***>
Date: Sunday, 3 April 2022 at 13:36
To: cgascoig/isctl ***@***.***>
Cc: Boris Aelen (baelen) ***@***.***>, Author ***@***.***>
Subject: Re: [cgascoig/isctl] Add support for expand (Issue #38)

Closed #38<https://github.com/cgascoig/isctl/issues/38>.

**Comment by baelen-git** [comment]
You are fantastic! Tried this out and works like a charm.
Even works with nested $expands.

```
❯ isctl get kubernetes clusterprofile -o yaml --select 'Name,NodeGroups' --expand 'NodeGroups($select=KubernetesVersion;$expand=KubernetesVersion($select=Version;$expand=Version($select=Name,Tags)))' | egrep -v "ObjectType|ClassId"
  Moid: 61af52617a6f722d31bd3d84
  Name: baelen-azure-arc
  NodeGroups:
    KubernetesVersion:
      Moid: 61f3bc387a6f722d31a3090a
      Version:
        Moid: 61f381e17a6f722d3093c093
        Name: 1.20.14-iks.0
        Tags:
        - Key: intersight.kubernetes.SupportStatus
          Value: UpgradeRequired
    Moid: 61af55bb7a6f722d31bdada4
    KubernetesVersion:
      Moid: 61f3bc387a6f722d31a3090a
      Version:
        Moid: 61f381e17a6f722d3093c093
        Name: 1.20.14-iks.0
        Tags:
        - Key: intersight.kubernetes.SupportStatus
          Value: UpgradeRequired
    Moid: 61af55cc7a6f722d31bdaf58
 ```
